lightkurve/targetpixelfile.py

```diff
@@ -550,7 +550,8 @@ def to_fits(self, output_fn=None, overwrite=False):
             output_fn = "{}-targ.fits".format(self.targetid)
         self.hdu.writeto(output_fn, overwrite=overwrite, checksum=True)
 
-    def interact(self, lc=None, notebook_url='localhost:8888', max_cadences=30000):
+    def interact(self, notebook_url='localhost:8888', max_cadences=30000,
+                startup_aperture=None, exported_filename=None):
```

Comment:
I think we'll want to name the parameter `aperture_mask` instead of `startup_aperture`,  to be consistent with other methods that take the same parameter?

Reply:
✅